showcase/src/assets/translations/fr.json

```diff
@@ -430,6 +430,49 @@
     "NEWS": {
       "TITLE": "Nouvelles"
     },
+    "NGX_FORM_ERRORS": {
+      "TITLE": "Reactive Forms errors",
+      "EXAMPLE": "Exemple",
+      "FORM": {
+        "VALIDATION": {
+          "REQUIRED": "{{fieldName}} est nécessaire",
+          "PASSWORD_REQUIRED": "{{fieldName}} doit être fourni",
+          "USER_NAME": {
+            "UNIQUE": "Votre nom d'utilisateur a déjà été pris"
+          },
+          "PASSWORD": {
+            "MAX_LENGTH": "Mot de passe ne peut pas y avoir plus de {{requiredLength}} caractères",
+            "MIN_LENGTH": "Mot de passe doit comporter au moins {{requiredLength}} caractères",
+            "PATTERN": "Votre mot de passe doit contenir au moins une majuscule, une minuscule et un chiffre."
+          },
+          "CONFIRM_PASSWORD": {
+            "ARE_EQUAL": "Non concordance des mots de passe passwords"
+          }
+        },
+        "HIDE_DETAILS": "Masquer les détails de validation",
+        "SHOW_DETAILS": "Afficher les détails de validation",
```

Comment:
Could you guys also have a look on these translations and tell me if they need to be corrected?